### PR TITLE
[Security] Add PHP attributes to a PHP annotations example

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: security.rst [hunk 1]
```diff
@@ -2199,30 +2199,55 @@ will happen:
...
+
+    .. code-block:: php-attributes
+
+        // src/Controller/AdminController.php
+        // ...
+
+        use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+
+        /**
+         * Require ROLE_ADMIN for *every* controller method in this class.
```
Comment (94noni) [inline security.rst]:
```suggestion
         * Require ROLE_ADMIN for *every* controller actions in this class.
```

what about this? perhaps the controller could have private methods
(same above)

File: security.rst [hunk 2]
```diff
@@ -2199,30 +2199,55 @@ will happen:
...
+         */
+        #[IsGranted('ROLE_ADMIN')]
+        class AdminController extends AbstractController
+        {
+            /**
+             * Require ROLE_ADMIN for only this controller method.
+             */
+            #[IsGranted('ROLE_ADMIN')]
```
Comment (94noni) [inline security.rst]: what about for the example on the action, using another role like `ROLE_SUPER_ADMIN`?

admin on whole controller, super admin on specific sub action

File: security.rst (hunk 1 above)
Comment (javiereguiluz) [reply security.rst]: Yes, I've reworded the message a bit.

File: security.rst (hunk 2 above)
Comment (javiereguiluz) [reply security.rst]: Good idea! I've changed this.

**Pull request merged**